src/lib.rs

```diff
@@ -207,8 +217,12 @@ impl SandboxFS {
         assert_eq!(fuse::FUSE_ROOT_ID, root.inode());
         nodes.insert(root.inode(), root);
 
+        #[cfg_attr(feature = "cargo-clippy", allow(redundant_field_names))]
```

Comment:
We might want to `#!` this at the crate level.

Reply:
Done. Added a first commit to do this.